### PR TITLE
KAA-356: Introduce resync configuration updates that are encoded using base configuration schema

### PR DESCRIPTION
**Pull request by ashvayka**

### CODE REVIEW

File: common/core/src/main/java/org/kaaproject/kaa/server/common/core/algorithms/delta/BaseBinaryDelta.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2014 CyberVision, Inc.
```
Comment (ghost) [inline common/core/src/main/java/org/kaaproject/kaa/server/common/core/algorithms/delta/BaseBinaryDelta.java]: Please, replace 2014 with 2014-2015.

File: server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/delta/DefaultDeltaService.java [hunk 2]
```diff
@@ -215,24 +217,29 @@ public DeltaCacheEntry compute(DeltaCacheKey deltaKey) {
...
-                                EndpointConfigurationDto endpointConfiguration = cacheService.getConfByHash(deltaKey.getEndpointConfHash());
-                                deltaCache = calculateDelta(endpointId, deltaCalculator, endpointConfiguration, mergedConfiguration);
-                            }
-                            if (cacheService.getConfByHash(deltaCache.getHash()) == null) {
-                                EndpointConfigurationDto newConfiguration = new EndpointConfigurationDto();
-                                newConfiguration.setConfiguration(deltaCache.getConfiguration());
-                                newConfiguration.setConfigurationHash(deltaCache.getHash().getData());
-                                cacheService.putConfiguration(deltaCache.getHash(), newConfiguration);
+                            
+                            if(deltaKey.isResyncOnly()){
```
Comment (ghost) [inline server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/delta/DefaultDeltaService.java]: Please, add code style spaces in if/else statement.

Comment (shpigor) [inline server/operations/src/main/java/org/kaaproject/kaa/server/operations/service/delta/DefaultDeltaService.java]: Code formatting

**Comment by shpigor** [comment]
Reviewed

**Pull request merged**